routes/Login.py

```diff
@@ -8,5 +8,9 @@ def __init__(self, path, methods) -> None:
     def getResponse(self, request: Request):
         if request.req_type == "POST":
             return Route.buildResponse()
+        elif request.req_type == "GET":
+            f = open("./public/login.html", "rb")
+            file = f.read()
```

Comment:
Encase in a try/ except block, 
`try:
      f =  open ...
      return ResponseBuilder stuff
except Exception:
      return response with 500 error code`